scss/custom.scss

```diff
@@ -1,253 +1,270 @@
 body {
```

Comment:
Please make sure that you don't reformat the entire file and only make the changes related to your PR!